tests/test_session.py

```diff
@@ -245,7 +247,6 @@ def test_document_sync_capabilities(self) -> None:
         self.assertEqual(session.text_sync_kind(), TextDocumentSyncKindIncremental)
         self.assertTrue(session.should_notify_did_change())
         self.assertFalse(session.should_notify_will_save())  # old-style text sync will never send willSave
-        self.assertFalse(session.should_request_will_save_wait_until())
         self.assertEqual(session.should_notify_did_save(), (False, False))
```

Comment:
If the lang server is "old" and has this in its server capabilities:

```
'capabilities': {
  'textDocumentSync': 2
}
```
Does `has_capability('textDocumentSync.change')` still return True?

Reply:
Not in this case since it doesn't include `change` property.

Reply:
Should it return True? The server can do incremental text sync, I think that's what one asks when querying `has_capability('textDocumentSync.change')`?

Does `has_capability` mean "Is this dotted path present" or does it mean "does the server have this capability" ?

Reply:
I'd say it's just "is this dotted path present and not False or Null".
This is also how previous implementation of `has_capability` worked really. We have more specific methods for capabilities checks that need more fancy logic.